gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsJmxMonitor.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
We should add tests even if this is a temporary solution.

Reply:
Here I need a little bit help and lead to set up the test correctly.

I added already
jvm.config and config.properties to activate /v1/jmx/mbean

But, I think I need to setup gateway config with monitorType JMX and backend settings with dummy user and pw etc.

@ebyhr or @willmostly Can you help me with this?